src/main/java/christmas/domain/order/Order.java

```diff
@@ -0,0 +1,42 @@
+package christmas.domain.order;
+
+import static christmas.domain.menu.MenuInfo.isDessert;
+import static christmas.domain.service.BeforeEventService.getMenuPrice;
+
+import christmas.domain.menu.MenuInfo;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Order {
+
+    private final LinkedHashMap<String, Integer> orderItems;
+
+    public Order() {
+        this.orderItems = new LinkedHashMap<>();
+    }
```

Comment:
생성과 orderItems 초기화를 동시에 해도 됩니다
```suggestion
    private final LinkedHashMap<String, Integer> orderItems = new LinkedHashMap<>();
```

Reply:
아하 넵넵 감사합니다~!